modules/installation-and-upgrade/pages/container-management/updating-proxy-containers.adoc

```diff
@@ -0,0 +1,26 @@
+= Update containers
+
+The {productname} {productnumber} Proxy containers running on [literal]``podman`` can be updated using the following command:
+----
+mgrpxy upgrade podman
+----
+
+Those running on a Kubernetes cluster can update using:
+
+----
+mgrpxy upgrade kubernetes
+----
+
+.Upgrading to specific version
+[NOTE]
+====
+If you do not specify the tag parameter , it will default to upgrading to the most recent version. To upgrade to a specific version, provide the tag parameter with the desired image tag.
```

Comment:
```suggestion
If you do not specify the tag parameter, it will default to upgrading to the most recent version. To upgrade to a specific version, provide the tag parameter with the desired image tag.
```